sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/ShareAsyncClient.java

```diff
@@ -1086,7 +1087,9 @@ public String generateSas(ShareServiceSasSignatureValues shareServiceSasSignatur
     }
 
     private Response<ShareStatistics> mapGetStatisticsResponse(SharesGetStatisticsResponse response) {
-        ShareStatistics shareStatistics = new ShareStatistics((int) (response.getValue().getShareUsageBytes() / 1024));
+        ShareStatistics shareStatistics =
+            new ShareStatistics((int) (response.getValue().getShareUsageBytes() / (Constants.GB)),
```

Comment:
Don't we want Math.ceil() of these values so we round up?

Reply:
It might also be better for the new constructor to just take a long and we calculate the share usage in bytes in the constructor.

Reply:
Done